inc/Dependencies/RocketLazyload/Assets.php

```diff
@@ -100,7 +100,28 @@ class_loaded: "lazyloaded",
 			$script = rtrim( $script, ',' );
 		}
 
-		$script .= '};';
+		$script .= '
+		},
+		{
+			elements_selector: ".rocket-lazyload",
```

Comment:
The value for the selector should be the one passed to the method as part of the `$args` instead of being hardcoded